.github/workflows/release-binaries.yaml

```diff
@@ -48,9 +48,13 @@ jobs:
       run: |
         set -x
 
-        # Install the latest version of hub
+        # Install the latest version of hub.
         ./build/install-hub.sh
 
-        # Create the release with all binaries
+        # Get a commit log since the previous tag.
+        previous_tag="$(git tag | grep -v '{{ github.ref }}' | grep '^v.*' | sort -Vr | head -1)"
+        changes="Changes:\n$(git log --pretty='format:* %s (by %an) on %cD' ${previous_tag}..HEAD -- internal/ cmd/ go.mod)"
```

Comment:
Remove the `*` from the formatting. It adds a lot of clutter. Use `-` instead if what you want to achieve is creating markdown bullet points.
Also consider changing the date format to something shorter:
```
changes="Changes:\n$(git log --pretty='format:- %s (by %an) on %cs%n' ${previous_tag}..HEAD -- internal/ cmd/ go.mod)"
```

Reply:
Done.

Reply:
Internal markdown format guide actually prefers asterisk. But - is fine too. go/g3doc-markdown#lists